.circleci/docker/build.sh

```diff
@@ -155,6 +157,22 @@ case "$image" in
     DB=yes
     VISION=yes
     ;;
+  pytorch-linux-bionic-py3.8-gcc9)
+    ANACONDA_PYTHON_VERSION=3.8
+    GCC_VERSION=9
+    PROTOBUF=yes
+    DB=yes
+    VISION=tes
```

Comment:
I assume this should be `yes` instead of `tes`? ;)

Reply:
Yes, you are right. Can you please fix the typo in PR you are working on.

Reply:
sure!